src/casync-tool.c

```diff
@@ -1657,6 +1682,7 @@ static int verb_extract(int argc, char *argv[]) {
         }
 
         (void) send_notify("READY=1");
+        extract_start_nsec = now(CLOCK_MONOTONIC);
```

Comment:
hmm, why maintain that as global concept? why not just do this internally in the ca_sync object? i.e. maybe just take a timestamp there when the first step is done, and then add another function called ca_sync_get_runtime() or so that returns the runtime of the sync object so far.

Reply:
Right, I didn't think about it. I reworked this part and added a `ca_sync_get_runtime()`. It's much better.